node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/acl/AclMaintainer.java

```diff
@@ -115,6 +131,31 @@ private boolean edit(NodeAgentContext context, String table, IPVersion ipVersion
         };
     }
 
+    void updateMetric(NodeAgentContext context, boolean updated, String ipVersion) {
+        Dimensions dimensions = generateDimensions(context);
+        if (!updated) {
+            metrics.declareGauge(Metrics.APPLICATION_NODE, ipVersion + METRIC_NAME_POSTFIX, dimensions, Metrics.DimensionType.PRETAGGED)
+                    .sample((System.currentTimeMillis() - lastSuccess.get(ipVersion)) / 1000);
```

Comment:
Consider introducing a variable for the metric value, and then simplify to one `declareGauge` call which avoids repeating all the other unchanged arguments.